src/Fable.Cli/CHANGELOG.md

```diff
@@ -31,6 +31,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * [GH-3649](https://github.com/fable-compiler/Fable/issues/3649) Fixes for `List.sortBy` (by @dbrattli)
 * [GH-3638](https://github.com/fable-compiler/Fable/issues/3638) Fixes for `Array.sort` and `Array.sortDescending` (by @dbrattli)
 * [GH-3655](https://github.com/fable-compiler/Fable/issues/3655) Fixes for Python output file names (by @dbrattli)
+* [GH-3660](https://github.com/fable-compiler/Fable/issues/3660) Fixes for decimal to string with culture (by @dbrattli)
```

Comment:
@dbrattli Should be placed under Unreleased section.

I updated the changelog in main